apiserver/app/createInvoice.go

```diff
@@ -86,6 +86,17 @@ func (a *App) createInvoice(res http.ResponseWriter, req *http.Request) error {
 	if err != nil {
 		return err
 	}
+	userId := context.GetUserId(req.Context())
+
+	if test {
```

Comment:
Also I'm thinking if this is the best place to check it. Cause counter will increase even though my invoice may have not been created (if there was some error later).
But we can keep it here and move around later